README.md

```diff
@@ -45,31 +45,31 @@ Fork and clone this repo:
 git clone git@github.com:[username]/malariagen-data-python.git
 ```
 
-Install Python 3.10, e.g.:
+Install Python 3.8 (current recommended version for local development), e.g.:
 
 ```bash
 sudo add-apt-repository ppa:deadsnakes/ppa
-sudo apt install python3.10 python3.10-venv
+sudo apt install python3.8 python3.8-venv
 ```
 
 Install pipx, e.g.:
 
 ```bash
-python3.10 -m pip install --user pipx
-python3.10 -m pipx ensurepath
+python3.8 -m pip install --user pipx
+python3.8 -m pipx ensurepath
 ```
 
 Install [poetry](https://python-poetry.org/docs/#installation), e.g.:
 
 ```bash
-pipx install poetry==1.4.1 --python=/usr/bin/python3.10
+pipx install poetry==1.4.1 --python=/usr/bin/python3.8
 ```
 
 Create development environment:
 
 ```bash
 cd malariagen-data-python
-poetry use 3.10
+poetry use 3.8
 poetry install
 ```
```

Comment:
Recommending Python 3.8 for local development, because otherwise if we use a later Python we can accidentally try to use features not supported in Python 3.8, which doesn't then crop up until CI fails. Better to fail fast locally.